RELEASE/scripts/autoscend.ash

```diff
@@ -2825,14 +2826,6 @@ void auto_begin()
 		}
 	}
 
```

Comment:
```suggestion
	}

	//This also should set our path too.
	string page = visit_url("main.php");
	page = visit_url("api.php?what=status&for=4", false);
```
restore a couple of deleted lines which should not have been deleted and are the reason this PR fails to validate

Reply:
unless the intent is to fix it by removing the subsequent references to page and just trust that mafia will correctly verify what ascension we are in

Reply:
Weird, it validated for me locally fine.